it/uk/gov/hmrc/agentsubscriptionfrontend/controllers/SignOutControllerISpec.scala

```diff
@@ -1,22 +1,54 @@
 package uk.gov.hmrc.agentsubscriptionfrontend.controllers
 
 import play.api.test.FakeRequest
-import play.api.test.Helpers.redirectLocation
-import play.api.test.Helpers._
+import play.api.test.Helpers.{redirectLocation, _}
+import uk.gov.hmrc.agentmtdidentifiers.model.Utr
+import uk.gov.hmrc.agentsubscriptionfrontend.models.KnownFactsResult
+import uk.gov.hmrc.agentsubscriptionfrontend.repository.KnownFactsResultMongoRepository
+
+import scala.concurrent.ExecutionContext.Implicits.global
 
 class SignOutControllerISpec extends BaseControllerISpec {
   private lazy val controller: SignedOutController = app.injector.instanceOf[SignedOutController]
 
   private lazy val logoutRedirectUrl = "/government-gateway-registration-frontend/choose-your-account"
```

Comment:
This `logoutRedirectUrl` val now looks unused as it has been replaced by `sosRedirectUrl`